gk/fib.c

```diff
@@ -1732,6 +1759,136 @@ del_fib_entry(const char *ip_prefix, struct gk_config *gk_conf)
 	return del_fib_entry_numerical(&prefix_info, gk_conf);
 }
 
+/*
+ * Stack when function starts:
+ *
+ * 5 |  gw_addrs  |
+ * 4 |  gt_addrs  |
+ * 3 |   gk_conf  | (unused in this function)
+ * 2 |    table   |
+ * 1 | prefix_str | (unused in this function)
+ *   |____________|
+ */
+static int
+read_grantor_lb_entries(lua_State *l, lua_Integer tbl_size)
+{
+	struct ipaddr *gt_addrs = (struct ipaddr *)lua_touserdata(l, 4);
+	struct ipaddr *gw_addrs = (struct ipaddr *)lua_touserdata(l, 5);
```

Comment:
Style: just have `gt_addrs` and `gw_addrs` as parameters of `read_grantor_lb_entries()`.